storage/src/metadata/storage.rs

```diff
@@ -28,9 +30,9 @@ pub struct Metadata<B: Blob, E: Clock + Storage<B> + Metrics> {
     _phantom_e: PhantomData<E>,
 }
 
-impl<B: Blob, E: Clock + Storage<B> + Metrics> Metadata<B, E> {
+impl<B: Blob, E: Clock + Storage<B> + Metrics, K: Array + Ord> Metadata<B, E, K> {
```

Comment:
I don't think you should need to add `Ord` here? (I think `Array` already implements this?)

Reply:
Yeah:
```
pub trait Array:
    Clone
    + Send
    + Sync
    + 'static
    + Eq
    + PartialEq
    + Ord
    + PartialOrd
    + Debug
    + Hash
    + Display
    + AsRef<[u8]>
    + Deref<Target = [u8]>
    + for<'a> TryFrom<&'a [u8], Error = <Self as Array>::Error>
    + for<'a> TryFrom<&'a Vec<u8>, Error = <Self as Array>::Error>
    + TryFrom<Vec<u8>, Error = <Self as Array>::Error>
    + SizedSerialize
    ```